### PR TITLE
63 alert crud

### PR DESCRIPTION
**Pull request by pratyushpal**
Updates some parts of Faq.js as well for consistency.

### CODE REVIEW

File: controllers/Alerts.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+	});
+});
+
+// editing an existing alert
+alertRouter.put('/edit', function(req, res) {
+	const alertToEdit = req.body;
+
+	Alert.findById(alertToEdit.id, function(err, alert) {
+		if (err) {
+			res.status(500).send(err);
```
Comment (theRoughCode) [inline controllers/Alerts.js:77]: Should return here

File: controllers/Alerts.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+		} else if (!alert) {
+			res.send('Alert ID not found!');
```
Comment (theRoughCode) [inline controllers/Alerts.js:79]: Should return here

File: controllers/Alerts.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+		res.json(mappedAlerts);
+	});
+})
+
+// get a specific alert
+alertRouter.get('/id/:id', function(req, res) {
+  const id = req.params.id;
+  Alert.findById(id, function(err, alert) {
+    if (err) {
+      return res.status(500).json(err);
```
Comment (theRoughCode) [inline controllers/Alerts.js:21]: Follow one-line statement from other files

**Review by theRoughCode — CHANGES_REQUESTED** [review]
A few minor formatting remarks.  Also, please update the wiki (https://github.com/uwblueprint/water-festival/wiki/API-Endpoints) and update the Postman JSON once this PR has been merged.

File: controllers/Alerts.js (hunk 3 above)
Comment (pratyushpal) [reply controllers/Alerts.js:21]: Is there a reason why there are some conditionals that aren't one-line statements in other files?

Comment (theRoughCode) [reply controllers/Alerts.js:21]: If the block statement related to a conditional has only one line, we want to have it without the braces

**Pull request closed without merging**